File: app/src/main/java/com/nicrosoft/consumoelectrico/realm/UUIDGenerator.java
package com.nicrosoft.consumoelectrico.realm;

import java.util.UUID;

/**
 * Created by Eder Xavier Rojas on 20/09/2016.
 */
public class UUIDGenerator {
    static String nextUUID() {
        return UUID.randomUUID().toString();
    }
}

File: app/src/main/java/com/nicrosoft/consumoelectrico/realm/UUIDGenerator.kt
package com.nicrosoft.consumoelectrico.realm

import java.util.*

/**
 * Created by Eder Xavier Rojas on 20/09/2016.
 */
object UUIDGenerator {
    @JvmStatic
    fun nextUUID(): String {
        return UUID.randomUUID().toString()
    }
}